pkgs/development/tools/profiling/oprofile/default.nix

```diff
@@ -12,10 +12,14 @@ stdenv.mkDerivation rec {
     sha256 = "0nn4wfvwy4nii25y6lwlrnzx9ah4nz0r93yk7hswiy6wxjs10wc4";
   };
 
-  buildInputs = [ binutils zlib popt pkgconfig ]
+  buildInputs = [ binutils zlib popt pkgconfig linuxPackages.kernelHeaders ]
     ++ stdenv.lib.optionals withGUI [ qt4 ];
 
   configureFlags = [
+      "--build ${stdenv.system}"
```

Comment:
Those flags are necessary? Are you sure? I thought that Nix `system` ids don't correspond to those used by autoconf.